rpc_jobs/unit/re_unit_tests.yml

```diff
@@ -49,15 +49,38 @@
             ]
           )
         },
-        "Single Use Slave": {
+        "Standard Slave Instance": {
           build(
-            job: "RE-unit-test-single-use-slave",
+            job: "RE-unit-test-slave-types",
             wait: true,
             parameters: [
               [
                 $class: 'StringParameterValue',
                 name: 'RPC_GATING_BRANCH',
                 value: env.RPC_GATING_BRANCH
+              ],
+              [
+                $class: 'StringParameterValue',
+                name: 'SLAVE_TYPE',
+                value: 'instance'
+              ]
```

Comment:
Ideally we'd want to unset this and allow it to use the default ("instance"), but we'd then need to add something into rpc_jobs/unit/single_use_slave.yml to actually validate we've gotten an instance, and not a container.  Know of any straight-forward way to do that?

Reply:
systemd-detect-virt?

Reply:
Brilliant :)

Reply:
Updated to do some actual assertions against the output of this.